src/python/artm/_js/ldavis.js

```diff
@@ -0,0 +1,1361 @@
+LDAvis = function(to_select, data) {
```

Comment:
There is no standard license: https://github.com/cpsievert/LDAvis/blob/master/LICENSE. Just a link to the copyright holder. :( 
I will ask the author.
Anyway, this code is just proof-of-concept and an impression for us. If there are problems with LDAvis permissions, we could easily make our original d3-based visualization.